ansys/dpf/core/custom_type_field.py

```diff
@@ -38,7 +38,7 @@ def __missing__(self, key):
 
 class CustomTypeField(_FieldBase):
     """Represents a simulation data container with each unitary data being of a custom type.
-    When initiliazing the CustomTypeField, a unitary data type should be given.
+    When initializing the ``CustomTypeField``, provide a unitary data type.
```

Comment:
```suggestion
    When initializing the ``CustomTypeField`` class, provide a unitary data type.
```
All literal strings need to have a noun after them.